lms/djangoapps/grades/course_grade_factory.py

```diff
@@ -152,7 +152,7 @@ def _read(user, course_data):
             course_data,
             persistent_grade.percent_grade,
             persistent_grade.letter_grade,
-            persistent_grade.passed_timestamp is not None,
+            persistent_grade.letter_grade is not u'',
```

Comment:
We have a `passed` property and `_compute_passed` method of `CourseGrade` object. Let's discuss how we can use this property and method to compute passed status at this point.